### DISCUSSION TITLE
Technikai segítség kérése

### DISCUSSION THREAD

**Discussion by migpeti** [nav-gov-hu/Online-Cash-Register-Logfile#39] [Q&A] [answered]
Üdvözlöm!

Elsősorban az a kérdésem, hogy technikai segítséget is lehet-e itt kérni, vagy arra van más fórum?
Nem szeretnék "felesleges" részleteket írni, ha itt nincs helye.
Röviden összefoglalva: A GenerateCashRegisterTestDataRequest kérésre kapok 500-as státuszú (Internal Server Error) választ, és nem tudok vele mit kezdeni. 
Egyelőre ennyi, további részletek is lesznek majd, ha jöhetnek.
Köszönöm!
Péter

**Comment by renced42** [comment, marked as answer]
Kedves @migpeti 

Issue-ban add fel a hibádat és megnézzük.
